windows/src/developer/TIKE/compile/MergeKeyboardInfo.pas

```diff
@@ -366,6 +390,19 @@ function TMergeKeyboardInfo.Failed(message: string): Boolean;
   Result := False;
 end;
 
+procedure TMergeKeyboardInfo.AddJSFile(const Filename: string);
```

Comment:
Merge this with `LoadJSFile`. The `AddJSFile` function is trivial and makes it sound like there may be more than one to add to the list.